tests/unittest/llmapi/apps/test_tool_parsers.py

```
@@ -5459,3 +5459,171 @@ def test_streaming_incremental_cutoff(self) -> None:
+            model="test-model",
+            messages=[{"role": "user", "content": "hello"}],
+            tools=sample_tools,
+        )
+        args = ChatPostprocArgs.from_request(req)
+        args.tool_parser = "qwen3"
+
+        # Malformed tool call that matches marker but cannot be extracted
+        text = "<tool_call>\n{invalid json\n</tool_call>"
+
+        with patch("tensorrt_llm.serve.postprocess_handlers.logger.warning") as mock_warn:
+            normal_text, calls = apply_tool_parser(
```

Ruff (0.16.2)

[warning] Unpacked variable `normal_text` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)